widgetssdk/src/main/java/com/glia/widgets/call/CallView.java

```diff
@@ -300,27 +299,24 @@ public void emitState(CallState callState) {
                     }
                     operatorStatusView.isRippleAnimationShowing(
                             callState.isCallNotOngoing() ||
-                                    callState.isCallOngoig()
+                                    callState.isCallOngoingAndOperatorIsConnecting()
                     );
 
-                    if (callState.hasMedia() && callState.callStatus.getOperatorProfileImageUrl() != null) {
+                    if (callState.isCallOngoingAndOperatorConnected() && callState.callStatus.getOperatorProfileImageUrl() != null) {
                         operatorStatusView.showProfileImage(
                                 callState.callStatus.getOperatorProfileImageUrl());
-                    } else if (callState.isCallOngoig() && callState.isCallOngoig() &&
+                    } else if (callState.isCallOngoingAndOperatorIsConnecting() && callState.isCallOngoingAndOperatorIsConnecting() &&
```

Comment:
Why is callState.isCallOngoingAndOperatorIsConnecting() called twice?

Reply:
good catch